src/transform/class/inheritance/UtilInherits.js

```diff
@@ -0,0 +1,147 @@
+import {matchesAst, extract} from '../../../utils/matchesAst';
+import {isVarWithRequireCalls} from '../../commonjs/importCommonjs';
+
+/**
+ * Processes nodes to detect super classes and return information for later
+ * transformation.
+ *
+ * Detects:
+ *   var util = require('util');
+ *   function Class1() {}
+ *   function Class2() {}
+ *   util.inherits(Class1, Class2);
+ */
+export default class UtilInherits {
+  /**
+   * @param {Object} cfg
+   *   @param {PotentialClass[]} cfg.potentialClasses
+   */
+  constructor({potentialClasses}) {
+    this.potentialClasses = potentialClasses;
+    this.utilNode = null;
+    this.inheritsNode = null;
```

Comment:
In Lebab repo we have a convention of using `undefined` instead of `null`. Unless there's a particular reason that requires the use of `null`, one should use `undefined` instead.